csclient/csclient_test.go

```diff
@@ -1051,7 +1042,7 @@ func (s *suite) TestMacaroonAuthorization(c *gc.C) {
 	body, err := json.Marshal([]string{"bob"})
 	c.Assert(err, gc.IsNil)
 
-	resp, err := s.client.DoWithBody(req, "/"+purl.Path()+"/meta/perm/read", httpbakery.SeekerBody(bytes.NewReader(body)))
+	resp, err := s.client.DoWithBody(req, "/"+rurl.URL.Path()+"/meta/perm/read", httpbakery.SeekerBody(bytes.NewReader(body)))
```

Comment:
This is a different thing purl == rurl.PreferredURL() In this case I suspect it doesn't matter.